src/os/JniSled.java

```diff
@@ -0,0 +1,35 @@
+package sh.tty.sled;
+
+import java.awt.image.BufferedImage;
+import java.awt.image.DataBufferInt;
+
+public class JniSled implements Runnable {
+    static {
+        System.loadLibrary("sled");
+    }
+
+    // only set these before starting the thread!!!
+    public static int MATRIX_X = 256;
```

Comment:
maybe you could set this via the constructor? if the size changes, you could just restart sled

Reply:
resolved in bd399dc596b3471d62559df35553a14df0717ec8